lib/foreman_docker/permitted_attributes.rb

```diff
@@ -0,0 +1,10 @@
+module Foreman
+  module PermittedAttributes
+    mattr_reader :container_attributes
+
+    @@container_attributes = [:command, :repository_name, :name, :compute_resource_id, :entrypoint,
```

Comment:
@unorthodoxgeek would you mind elaborating on why [this change](https://github.com/komidore64/foreman-docker/commit/ff1150ccad9e19becde013341e4b0cef17ffbeed) was necessary?